site/formulas/decoding.md

```diff
@@ -1 +1,39 @@
 # Decoding to Extended Coordinates
+
+Given 32 octets of data, `s_bytes`, representing a compressed
```

Comment:
If these formulas are not `ristretto255`-specific, then it may not be 32 bytes of data.  Maybe it would be best to separate the `ristretto255`-specific parts from the general parts, so that the formulas can be implemented for any `a, d`, and then have notes that are specific to the `ristretto255` case?

Then "step 1" would be something like "check that `s_bytes` is a canonical encoding of a field element; see below for notes on how to do this for `ristretto255`".

Reply:
I'm not sure that someone is going to care about the generalised implementation details with a different curve in mind, or rather, if they were, they'll likely be looking at the maths notes and not these developer-focused notes? So maybe we should be explicit about this being for `ristretto255`?

Reply:
I think that it's good to be explicit about what parts are `ristretto255`, both because it's conceptually cleaner, and because it means that if someone tries to implement Ristretto with a different curve (e.g., JubJub) they can check against these formulas.